tests/components/evohome/snapshots/test_climate.ambr

```diff
@@ -0,0 +1,379 @@
+# serializer version: 1
+# name: test_zon_set_hvac_mode[default]
+  list([
+    tuple(
+      dict({
+        'HeatSetpointValue': 5.0,
+        'setpointMode': <ZoneMode.PERMANENT_OVERRIDE: 'PermanentOverride'>,
+      }),
+    ),
+  ])
+# ---
+# name: test_zon_set_hvac_mode[h032585]
+  list([
+    tuple(
+      dict({
+        'HeatSetpointValue': 4.5,
+        'setpointMode': <ZoneMode.PERMANENT_OVERRIDE: 'PermanentOverride'>,
+      }),
+    ),
+  ])
+# ---
+# name: test_zon_set_hvac_mode[h099625]
```

Comment:
Seems some remains from the old naming are still in the snapshot

Reply:
Woops - forgot to delete before `--snapshot-update`